docs/notebooks/summarization_tutorial.ipynb

```diff
@@ -409,25 +409,102 @@
     "\n",
     "Another possible reason for the difference in running times is that the problems converge at different rates, meaning that the error drops slower for some datasets than for others.\n"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Montemurro and Zanette's entropy based keyword extraction algorithm\n",
+    "\n",
+    "[This paper](https://arxiv.org/abs/0907.1558) describes a technique to identify words that play a significant role in the large-scale structure of a text. These typically correspond to the major themes of the text. The text is divided into blocks of ~1000 words, and the entropy of each word's distribution amongst the blocks is\n",
+    "caclulated and compared with the expected entropy if the word were distributed randomly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "from gensim.summarization import mz_keywords"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "text=requests.get(\"http://www.gutenberg.org/files/49679/49679-0.txt\").text\n",
+    "mz_keywords(text,scores=True,threshold=0.005)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, the algorithm weights the entropy by the overall frequency of the word in the document. We can remove this weighting by setting weighted=False"
```

Comment:
Where is the output of your code? descriptions are needed too of course, but user want's to see the result (how it works).